docker/docker-compose.yml

```diff
@@ -1,7 +1,23 @@
 version: '3'
 services:
+  local-storage-service-database:
```

Comment:
add volume mounting here for future container updates without data loss

Reply:
added